### DISCUSSION TITLE
Using Shadcn-ui with Inertia and AdonisJS

### DISCUSSION THREAD

**Discussion by giscus** [matfire/Portfolio#17] [Announcements]
# Using Shadcn-ui with Inertia and AdonisJS

Matteo Gassend is a developer with 4 of experience

https://matteogassend.com/blog/using-shadcn-ui-with-inertia-and-adonisjs

**Comment by Nallanos** [comment]
Merci mille fois, tu m'as economisé tellement de tellement de temps !
